From b7290560082e91673431de79e1fa318c9fd90261 Mon Sep 17 00:00:00 2001
From: Danomi Manchego <danomimanchego123@gmail.com>
Date: Sat, 25 Oct 2014 19:42:38 +0200
Subject: [PATCH 1/5] log4c.m4: fix "underquoted definition of AM_PATH_LOG4C"
 warning

When autoreconfiguring liblog4c-localtime, there is a warning from
autoconf caused by an underquoted definition of AM_PATH_LOG4C. This
patch fixes this warning.

Submitted upstream: https://github.com/rcmadruga/log4c-localtime/pull/1

Signed-off-by: Danomi Manchego <danomimanchego123@gmail.com>
Signed-off-by: Thomas Petazzoni <thomas.petazzoni@free-electrons.com>
---
 log4c.m4 | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/log4c.m4 b/log4c.m4
index 551a90d..96424c0 100644
--- a/log4c.m4
+++ b/log4c.m4
@@ -4,7 +4,7 @@
 dnl AM_PATH_LOG4C([MINIMUM-VERSION, [ACTION-IF-FOUND [, ACTION-IF-NOT-FOUND]]])
 dnl Test for LOG4C, and define LOG4C_CFLAGS and LOG4C_LIBS
 dnl
-AC_DEFUN(AM_PATH_LOG4C,
+AC_DEFUN([AM_PATH_LOG4C],
 [dnl 
 dnl Get the cflags and libraries from the log4c-config script
 dnl
-- 
2.0.0

